sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/AccountSasImplUtil.java

```diff
@@ -41,19 +41,37 @@
 
     private String resourceTypes;
 
+    private String encryptionScope;
+
     /**
      * Creates a new {@link AccountSasImplUtil} with the specified parameters
      *
      * @param sasValues {@link AccountSasSignatureValues}
      */
     public AccountSasImplUtil(AccountSasSignatureValues sasValues) {
+        this(null, sasValues);
+    }
+
+    /**
+     * Creates a new {@link AccountSasImplUtil} with the specified parameters
+     *
+     * @param encryptionScope An encryption scope that will be applied to any write operations performed with the sas
+     * @param sasValues {@link AccountSasSignatureValues}
+     */
+    public AccountSasImplUtil(String encryptionScope, AccountSasSignatureValues sasValues) {
```

Comment:
we don't need extra ctor. please remove.

Reply:
done